portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,6 +38,12 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
See [my comment](https://github.com/mmurray22/my-portfolio/pull/21#discussion_r451219463) on the parent CL

Reply:
will be addressed in the merge! should I change on this branch too?

Reply:
yeah - i think anytime you update a parent branch you should "evolve" or merge the changes into all of its children:

- this will catch merge conflicts earlier
- will make the PRs easier to review and reason about

Reply:
k will do!